test/e2e/windows/hybrid_network.go

```diff
@@ -79,6 +80,58 @@ var _ = SIGDescribe("Hybrid cluster network", func() {
 
 		})
 
+		ginkgo.It("should have stable networking for Linux and Windows with services", func() {
```

Comment:
we could be explicit on `with clusterIP services`. 
I know there's one log above, but nothing denies the test to change the service type later for example.

Reply:
yeah, be explicit, most probably you'll have to add a new one for nodeport later :smile:

Reply:
Makes sense!